tests/WE2E/test_configs/special_builds/config.UFS_FIRE.yaml

```diff
@@ -0,0 +1,60 @@
+metadata:
+  description: >-
+    Tests the UFS_FIRE capability
+user:
+  RUN_ENVIR: community
+workflow:
+  CCPP_PHYS_SUITE: FV3_HRRR
+  PREDEF_GRID_NAME: SUBCONUS_CO_3km
+  DATE_FIRST_CYCL: '2020081318'
+  DATE_LAST_CYCL: '2020081318'
+  FCST_LEN_HRS: 6 # 27 hours of LBCs staged on Derecho
+  PREEXISTING_DIR_METHOD: rename
+task_get_extrn_ics:
+  USE_USER_STAGED_EXTRN_FILES: true
+  EXTRN_MDL_NAME_ICS: HRRR
+  FV3GFS_FILE_FMT_ICS: grib2
+  EXTRN_MDL_FILES_ICS:
+    - '{fyyyymmdd}.hrrr.t{fhh}z.wrfprsf00.grib2'
+task_get_extrn_lbcs:
+  USE_USER_STAGED_EXTRN_FILES: true
+  EXTRN_MDL_FILES_LBCS:
+    - '{fyyyymmdd}.hrrr.t{fhh}z.wrfprsf00.grib2'
+  EXTRN_MDL_NAME_LBCS: HRRR
+  EXTRN_MDL_LBCS_OFFSET_HRS: 0
+  LBC_SPEC_INTVL_HRS: 3
+  FV3GFS_FILE_FMT_LBCS: grib2
+task_run_fcst:
+  QUILTING: true
+  PRINT_ESMF: True
+  OMP_NUM_THREADS_RUN_FCST: 1
+task_plot_allvars:
+  COMOUT_REF: ""
+fire:
+  UFS_FIRE: True
+  FIRE_NUM_TASKS: 1
+  FIRE_INPUT_DIR: /glade/derecho/scratch/kavulich/FIRE/2024/test_data
```

Comment:
I assume this would eventually live under the EPIC role account space on the Tier I platforms?

Reply:
Yes, I believe EPIC is working on staging the data in the appropriate locations; I will update the test file as necessary when that is complete.